src/box-client.ts

```diff
@@ -13,19 +12,22 @@
  * A collaboration role constant
  * @typedef {string} CollaborationRole
  */
+type CollaborationRole = string;
 
 /**
-* A Box file or folder type constant
-* @typedef {string} ItemType
-*/
+ * A Box file or folder type constant
+ * @typedef {string} ItemType
+ */
+type ItemType = string;
```

Comment:
What are your thoughts on making this more specific, like `type ItemType = 'file' | 'folder'`?